.gitignore

```diff
@@ -1,2 +1,3 @@
 *.egg-info
 __pycache__
+.vscode
```

Comment:
Don't add non-project files here please, for local IDE stuff use a global ignore file or `.git/info/exclude`

https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer